ddclient.in

```diff
@@ -4238,6 +4259,174 @@ sub nic_dnsexit_update {
     }
 }
 ######################################################################
+## nic_dnsexit2_examples
+######################################################################
+sub nic_dnsexit2_examples {
+    return <<"EoEXAMPLE";
+o 'dnsexit2'
+
+The 'dnsexit2' protocol is the new API protocol used by the dynamic hostname services
+of the 'DNSExit' dns services. This is currently used by the free
+dynamic DNS service offered by www.dnsexit.com.
+
+Configuration variables applicable to the 'dnsexit2' protocol are:
+  protocol=dnsexit2         ##
+  password=YourAPIKey       ## API Key of your account.
+  server=api.dnsexit.com    ## defaults to api.dnsexit.com.
+  path=/dns/                ## defaults to /dns/.
+  record-type=A             ## defaults to A record.
+  ttl=5                     ## defaults to 5 minutes.
```

Comment:
This is the record TTL? Can we somehow leave this unchanged?

Reply:
Correct, this is the record TTL. Good question on whether filling TTL is required, I have not found a clear definition and have relied on the examples from the documentation that all provide TTL in the update request [(docs)](https://dnsexit.com/dns/dns-api/#example-update-domain-ip). But probably you could omit it. However some users might want to set it anyway, I think code is simpler to just send it along, in line with the docs?